src/i18n/fr.json

```diff
@@ -41,6 +41,7 @@
   "PROFILE_SETUP_ADDRESS": "Addresse",
   "PROFILE_SETUP_TERMS_CONFIRM": "J'ai lu et j'accepte les conditions de service",
   "PROFILE_SETUP_TERMS_READ": "Lire les conditions de service",
+  "PROFILE_SETUP_POlICY_READ": "",
```

Comment:
Never leave empty a translation. It won't display anything in that case.

Just don't define that string and the fallback will be english!